### ISSUE TITLE
load skope analysis currently broken

### ISSUE THREAD

**Issue by alee** [openskope/skopeui#172]
a request.json like this:

```
{
    "dataset_id": "paleocar_v3",
    "variable_id": "ppt_water_year",
    "selected_area": {
        "type": "Feature",
        "properties": {},
        "geometry": {
            "type": "Polygon",
            "coordinates": [
                [
                    [-108.074913, 36.01764],
                    [-108.077316, 36.046787],
                    [-108.039894, 36.046509],
                    [-108.039551, 36.075091],
                    [-108.021011, 36.075187],
                    [-108.021011, 36.089517],
                    [-107.903938, 36.088962],
                    [-107.903595, 36.084621],
                    [-107.894669, 36.084801],
                    [-107.895012, 36.044844],
                    [-107.840767, 36.045122],
                    [-107.840767, 35.978006],
                    [-107.905312, 36.012728],
                    [-107.900505, 36.015974],
                    [-107.967796, 36.016807],
                    [-107.966766, 36.023748],
                    [-107.976379, 36.023696],
                    [-107.976379, 36.031521],
                    [-108.038864, 36.031521],
                    [-108.039207, 36.017084],
                    [-108.074913, 36.01764]
                ]
            ]
        }
    },
    "time_range": {
        "gte": "1033-01-01",
        "lte": "2000-01-01"
    },
    "zonal_statistic": "mean",
    "transform": {
        "type": "ZScoreFixedInterval"
    },
    "requested_series_options": [{
        "name": "Transformed",
        "smoother": {
            "type": "NoSmoother"
        }
    }, {
        "name": "Smoothed",
        "smoother": {
            "type": "MovingAverageSmoother",
            "method": "centered",
            "width": 11
        }
    }],
    "max_processing_time": 8000
}
```

went to a page with a base temporal range (for paleocar v3, 103-2000)

**Comment by alee** [comment]
there's some inconsistency in initialization of components and watched variables that needs to be sorted out

**Comment by alee** [comment]
is hopefully fixed after 5eb4b19568a5507c0f4e960ed1bd5a369be57b53
